Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Components/Pages/Home.razor

```diff
@@ -1,104 +1,133 @@
 @page "/"
 
-@using Amazon.Lambda.TestTool.Services
 @using Amazon.Lambda.TestTool.Models
-@using Amazon.Lambda.TestTool.SampleRequests;
-@using Amazon.Lambda.TestTool.Services.IO
-@using Microsoft.AspNetCore.Http;
-
-@inject IHttpContextAccessor HttpContextAccessor
-@inject IRuntimeApiDataStoreManager DataStoreManager
-@inject IDirectoryManager DirectoryManager
-@inject IThemeService ThemeService
-@inject IJSRuntime JsRuntime
+@using Amazon.Lambda.TestTool.SampleRequests
+@using Amazon.Lambda.TestTool.Services
 
 <PageTitle>Lambda Function Tester</PageTitle>
 
-<h3>Lambda Function Tester</h3>
+<div class="d-flex align-items-center mb-2">
+    <h3 class="me-auto">Lambda Function Tester</h3>
+    <div class="form-floating @(_availableLambdaFunctions.Count <= 1 ? "d-none" : "")" style="min-width: 300px;">
+        <select class="form-select" id="selectedFunctionName" @bind="SelectedFunctionName">
```

Comment:
`No function registered or default function registered` - in this case for default function registered - Is the picture in the PR for this correct? Based on the code it seems like it will still show the drop down with "default lambda" as the value, which i think should be the correct behavior. But i just wanted to double check.

For no function registered - wondering if we should still show the dropdown menu, but then have an error/message saying a function needs to be registered or something to let the user know?


same thing with no function registered - wondering if we should tell the user "No functions registered and you need to register one.. " (or maybe have the drop down there but empty to show that it wont work?) or something like that.

Reply:
Functionally, having the default function registered or no function registered, they are the same from a UI perspective. I don't want to show the dropdown because it serves no purpose. I want it to behave similar to how the old test tool behaves. As soon as another function is registered, the drop down will pop-up and allow the users to switch. In the code, `@(_availableLambdaFunctions.Count <= 1 ? "d-none" : "")` the drop down will disappear if _availableLambdaFunctions has 1 function or less. This is the default case, since at least the default function will always get registered.